src/skmultiflow/meta/learn_nse.py

```diff
@@ -1,8 +1,9 @@
 import copy as cp
 import numpy as np
-from skmultiflow.core.base import StreamModel
 from sklearn.tree import DecisionTreeClassifier
 
+from skmultiflow.core.base import StreamModel
```

Comment:
This change is unnecessary, can you keep the original imports? Just for clarity.

Reply:
It was automatically reformated by my editor.
In PEP-8 you can read https://www.python.org/dev/peps/pep-0008/#imports
```

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.
```

I think its good change

Reply:
I see your point and agree. I will merge the PR.

Thanks for the help @bgulowaty